docs/data/joy/main-features/color-inversion/ColorInversionFooter.tsx

```diff
@@ -28,14 +27,14 @@ export default function ColorInversionFooter() {
       color={color}
       invertedColors
       sx={{
-        ...(color !== 'warning' && {
+        ...(color !== 'neutral' && {
           bgcolor: `${color}.800`,
         }),
         flexGrow: 1,
```

Comment:
Could we add 
```js
…color === 'warning' && {
  bgcolor: 'warning.400',
}
```

Reply:
Tried locally and it didn't seem to have any effect 😞

Reply:
For the sake of alignment: I'll merge this PR as soon as it is approved due to the various copywriting & design tweaks and then we can go back to this issue as it seems something isolated 😬